aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsFluentClientDecorator.kt

```diff
@@ -103,13 +103,24 @@ class AwsFluentClientDecorator : ClientCodegenDecorator {
         baseGenerator.protocolSupport,
         baseGenerator.operationShape,
         renderClientCreation = { params ->
+            rust("let mut ${params.configBuilderName} = ${params.configBuilderName};")
+            if (codegenContext.smithyRuntimeMode.defaultToOrchestrator) {
+                // TODO(enableNewSmithyRuntimeLaunch): A builder field cannot be accessed directly in the orchestrator mode.
```

Comment:
https://github.com/awslabs/smithy-rs/pull/2792 adds getters to the builders, so that should unblock this comment

Reply:
Thanks for letting me know. Updated the comment c41a167 accordingly.